src/appsignal/cli.py

```diff
@@ -1,2 +1,118 @@
+from abc import ABC, abstractmethod
+import os
+
+from typing import Optional
+from docopt import docopt
+
+from .__about__ import __version__
+
+DOC = """AppSignal for Python CLI.
+
+Usage:
+  appsignal install [--push-api-key=<key>]
+  appsignal (-h | --help)
+  appsignal --version
+
+Options:
+  -h --help             Show this screen and exit.
+  --version             Show version number and exit.
+  --push-api-key=<key>  Push API Key to use for the installation.
+"""
+
+
 def run():
-    print("Hello AppSignal!")
+    version = f"AppSignal for Python v{__version__}"
+
+    arguments = docopt(DOC, version=version)
+
+    return command_for(arguments).run()
+
+
+class CLICommand(ABC):
+    @abstractmethod
+    def run(self) -> int:
+        raise NotImplementedError
+
+
+def command_for(arguments) -> CLICommand:
+    if arguments["install"]:
+        return InstallCommand(push_api_key=arguments["--push-api-key"])
+    else:
+        raise NotImplementedError
+
+
+INSTALL_FILE_TEMPLATE = """from appsignal import Appsignal
+
+appsignal = Appsignal(
+    active=True,
+    name="{name}",
+    push_api_key="{push_api_key}",
+    environment="development",
+)
+"""
+
+INSTALL_FILE_NAME = "appsignal_config.py"
+INSTALL_FILE_PATH = f"{os.getcwd()}/{INSTALL_FILE_NAME}"
+
+
+class InstallCommand(CLICommand):
+    _push_api_key: Optional[str]
+    _name: Optional[str]
+
+    def __init__(self, push_api_key=None):
+        self._push_api_key = push_api_key
+        self._name = None
+
+    def run(self) -> int:
+        print("Welcome to the AppSignal for Python installer!")
+        self._input_name()
+
+        if self._push_api_key is None:
+            self._input_push_api_key()
```

Comment:
Eventually we should validate this key's validity and exit if it's not. We can create a new issue for that and pick it up later. I don't see it as a blocker.

Reply:
Ah, yes! I may give it a try now. If it's too hard then I'll create an issue. But I don't think it will be hard because Requests is wonderful.

Reply:
I don't quite understand that. How can the installer support those if we don't have a config yet? (I suppose if given as environment variables?)

Anyway, ETOOHARD: https://github.com/appsignal/appsignal-python/issues/42